required-version.go

```diff
@@ -0,0 +1,8 @@
+//go:build !go1.16
```

Comment:
This information can be put into the go.mod to stop building unless it has the correct version of go.

Reply:
That is true, I'll test it when I'm back